src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+    
+	private String block;
+    
+	public Block(String block) {
+		this.block = block;
+	}
+    
+	public String getBlock(){
+		return this.block;
+	}
+}
```

Comment:
This class (and the other 3 new classes) are a little bare bone :-).

Consider adding simple validation method to them.